docs/release-notes/2.43.0.md

```diff
@@ -28,6 +28,8 @@ This release introduces this and that
 
 Additionally, the endpoint now includes a customizable line-count parameter in the download URL, allowing you to specify a custom number of log lines or opt for unlimited log size. For more information, see the [API documentation](https://percona-pmm.readme.io/reference/logs).
 
+- [PMM-13159](https://perconadev.atlassian.net/browse/PMM-13159) - Removed experimental mongodb collection details dashboard as the same info is now available on the new experimental mongodb cluster summary dashboard.
```

Comment:
I wonder if we should spell it as `MongoDB`?

Reply:
Agree, just checked older notes, dashboard names are written as MongoDB Collection Details. ill update the same